components/input/src/auro-input.js

```diff
@@ -50,6 +50,18 @@ export class AuroInput extends BaseInput {
     this.helpTextTag = versioning.generateTag('auro-formkit-input-helptext', helpTextVersion, AuroHelpText);
   }
 
+  static get properties() {
+    return {
+
+      /** Exposes inputmode attribute for input.  */
+      inputmode: {
+        type: String,
+        attribute: true,
+        reflect: true
+      }
+    };
```

Comment:
to match this with `base-input.js`'s prop.
then we do not change line 149

```suggestion
      /** Exposes inputmode attribute for input.  */
      inputMode: {
        type: String,
        attribute: 'inputmode",
        reflect: true
      }
    };
```